apps/explorer-e2e/README.md

```diff
@@ -21,14 +21,19 @@ The e2e tests run against a locally running instance of the Vega network, manage
 - Install the required Vega smart contracts
 - Set up DataNodes with a running GraphQL and REST APIs.
 
-Refer to the [Vega Capsule readme](https://github.com/vegaprotocol/vegacapsule#readme) for setting up and running Capsule. You will need [Go 1.19 or later](https://go.dev/doc/install) and [Docker](https://docs.docker.com/get-docker/) installed.
+1. Refer to the [Vega Capsule readme](https://github.com/vegaprotocol/vegacapsule#readme) for setting up and running Capsule - follow by Pre-start and Quick Start (points 1-2)
+2. Bootstrap with auto-installed dependencies including wallet
+
+```bash
+vegacapsule network bootstrap --config-path=../frontend-monorepo/vegacapsule/config.hcl --force
+```
 
 ### Troubleshooting
 
 - You may need to run `vegacapsule nodes unsafe-reset-all` to get a clean network state
 
 ## Vega Wallet Setup
 
-Start by [downloading the Vega wallet software here](https://github.com/vegaprotocol/vega/releases).
+You can then refer to (or run) `frontend-monorepo/vegacapsule/setup-vegawallet.sh`. This will initialise and configure your wallet to have the correct public keys and network config to run against capsule.
 
-You can then refer to (or run) `vegacapsule/setup-vegawallet.sh`. This will initialise and configure your wallet to have the correct public keys and network config to run against capsule.
+Go to `.env` file in explorer-e2e folder and paste there public key for `CYPRESS_VEGA_WALLET_API_TOKEN`
```

Comment:
chore (blocking): VEGA_WALLET_API_TOKEN should contain the long lived api token, not the public key

Same in the other readme too.

Reply:
Also I think the following is clearer and more grammatically correct:

Go to the `.env` file in `apps/explorer-e2e` and set the CYPRESS_VEGA_WALLET_API_TOKEN environment variable by pasting in your wallets long lived api token